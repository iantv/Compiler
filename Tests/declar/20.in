struct point{
	int x;
	int y;
}var1;