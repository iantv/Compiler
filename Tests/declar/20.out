struct point
	x: variable
		int
	y: variable
		int
var1: variable
	struct point
		x: variable
			int
		y: variable
			int
